### PR TITLE
Merge dev in main :sparkles:

### PR DESCRIPTION
**Pull request by o-laurent**
- [x] Update packages :zap:
- [x] Remove useless functional transforms :fire:
- [x] Add a link to Packed-Ensembles medium post :book:
- [x] Add ImageNet variations :sparkles:
- [x] Don't compute coverage on datasets and load_pretrained methods
- [x] Merge baselines simplifications PR :hammer:
- [x] Add tests (missing for transforms & pixmix)
- [x] Add update package procedure

### CODE REVIEW

**Comment by o-laurent** [comment]
@alafage I am ready to merge as soon as we finish #25

File: torch_uncertainty/models/resnet/masked.py [hunk 1]
```diff
@@ -279,7 +279,8 @@ def _make_layer(
         return nn.Sequential(*layers)
 
     def forward(self, x: Tensor) -> Tensor:
-        out = F.relu(self.bn1(self.conv1(x)))
+        out = x.repeat(self.num_estimators, 1, 1, 1)
```
Comment (o-laurent) [inline torch_uncertainty/models/resnet/masked.py:282]: Is it normal to make 2 repeats (cf. ClassificationEnsemble)?

Comment (o-laurent) [reply torch_uncertainty/models/resnet/masked.py:282]: (The input is not repeated in ClassificationEnsemble, my mistake.)

**Comment by o-laurent** [comment]
The current code is acceptable to me for merging.

File: tests/layers/test_batched_layers.py [hunk 2]
```diff
@@ -42,6 +43,7 @@ class TestBatchConv2d:
 
     def test_conv_one_estimator(self, img_input: torch.Tensor):
         layer = BatchConv2d(6, 2, num_estimators=1, kernel_size=1)
+        print(layer)
```
Comment (alafage) [inline tests/layers/test_batched_layers.py:46]: Is this print() really necessary?

**Review by alafage — APPROVED** [review]
I believe the state of this pull request is fine enough to be merged.

File: tests/layers/test_batched_layers.py (hunk 2 above)
Comment (o-laurent) [reply tests/layers/test_batched_layers.py:46]: Yes, to go though extra repr

Comment (alafage) [reply tests/layers/test_batched_layers.py:46]: fair enough
